src/app/utils/formValidationUtils.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+function isDate(input, minYear = 1902, maxYear = new Date().getFullYear()) {
+  // regular expression to match required date format
+  const regex = /^(\d{2})\/(\d{2})\/(\d{4})$/;
+
+  if (input === '') {
+    return false;
+  }
+
+  if (input.match(regex)) {
+    const temp = input.split('/');
+    const dateFromInput = new Date(`${temp[2]}/${temp[0]}/${temp[1]}`);
+
+    return (
+      dateFromInput.getDate() === Number(temp[1])
+      && (dateFromInput.getMonth() + 1) === Number(temp[0])
+      && Number(temp[2]) > minYear
+      && Number(temp[2]) < maxYear
+    );
+  }
+
+  return false;
+}
+
+/**
+ * createAnchorID(wholeText)
+ * Splits the error message into two parts. One will be wrapped by an <a> tag, and another will not.
+ *
+ * @param {string} wholeText
+ * return {object} {anchorText, restText}
+ */
+function createAnchorText(wholeText) {
+  const anchorText = (wholeText && typeof wholeText === 'string') ?
+    wholeText.split(' field')[0] : '';
+  const restText = (!anchorText) ? '' : ` field ${wholeText.split(' field')[1]}`;
+
+  return { anchorText, restText };
+}
+
+/**
+ * createAnchorID(key)
+ * Creates the proper anchor ID for the href of the <a> tag in the error message.
+ *
+ * @param {string} key
+ * return {string}
+ */
+function createAnchorID(key) {
+  let hashElement = (key && typeof key === 'string') ?
+    `${key.charAt(0).toUpperCase()}${key.substr(1)}` : '';
+
+  if (hashElement === 'DateOfBirth') {
```

Comment:
I just copied these from GALC. They're not being used but just copied in case they could be useful. So don't worry about these being right or not. They can be updated later as we know the actual requirements.